### PR TITLE
reworked documentation side-nav to add sublevel

### PR DESCRIPTION
**Pull request by thimy**
image

Not sure about this appearance, feel free to comment!

### CODE REVIEW

**Comment by noirbizarre** [comment]
Can you add some screenshots ?

**Comment by noirbizarre** [comment]
OK, so in this screenshot **Menu #1 > Another sub-menu #3** is active (ie. is the current page) and **Menu #1 > Sub-menu #2** is hovered.

Can you provide some screenshots for the following cases ?
- **Menu #1** is active
- **Menu #1** is not active and collapsed but hovered
- Same as current screenshot but with **Menu #1 > Another sub-menu #3** hovered
- Same as current screenshot but with **Menu #1** hovered

Currently, there seem to be almost the same visual code for parent menu expanded (plain blue background) and the hovered item (plain gray background). It a big deal but hard to see if this is coherent without the case these other cases

**Comment by thimy** [comment]
*Menu #1* is active
image

*Menu #1* is collapsed and hovered (unfolds on click I guess)
image

*Menu #1* active, hover on *submenu #3*
image

*Menu #1* active and hovered
image

**Comment by noirbizarre** [comment]
Seems fine, but a detail bother me: the right arrow seems to be visible only on hover which make the submenu hard to discover.
Can we make it permanent ?

File: src/css/documentation.css [hunk 1]
```diff
@@ -9,7 +9,7 @@
   border-right: 1px solid var(--theme-border-lighter);
 }
 
-.side-pane ul {
+.side-pane__menu {
```
Comment (noirbizarre) [inline src/css/documentation.css]: I think this one is lacking its `.side-pane` namespace

**Comment by thimy** [comment]
I actually added the arrows afterwards and didn't update the screenshot, should be looking like this now:
image

**Pull request merged**